Library/Homebrew/extend/ENV/super.rb

```diff
@@ -28,8 +28,7 @@ def self.extended(base)
   def self.bin
     return unless MacOS.has_apple_developer_tools?
 
-    bin = HOMEBREW_ENV_PATH.subdirs.reject { |d| d.basename.to_s > MacOS::Xcode.version }.max
-    bin.realpath unless bin.nil?
+    HOMEBREW_ENV_PATH.subdirs.max.realpath
```

Comment:
> However, since the scripts therein work with xcrun and xcode-select, they are probably not as useful as platform-agnostic wrappers and maybe shouldn't be pulled in at all.

👍